dgraphtest/local_cluster.go

```
@@ -864,3 +869,44 @@ func (c *LocalCluster) inspectContainer(containerID string) (string, error) {
 	}
 	return string(raw), nil
 }
+
+func (c *LocalCluster) GeneratePlugins(raceEnabled bool) {
```

this function should return an error